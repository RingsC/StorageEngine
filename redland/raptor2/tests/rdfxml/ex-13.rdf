<?xml version="1.0" encoding="utf-8"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns:play="http://somewhere.example.com/"
         xmlns:enc="http://somewhere.example.com/">
   <!-- Check this doesn't crash anymore - parseType="Resource" -->

   <!-- Should generate 3 statements as described in ex-13.out -->

<rdf:Description rdf:about="http://www.w3.org/">
   <play:xml-cannonicalized rdf:parseType="Resource">
      <enc:hash-sha-1 rdf:parseType="Resource">
         <enc:base64>jd8734djr08347jyd4</enc:base64>
      </enc:hash-sha-1>
   </play:xml-cannonicalized>
</rdf:Description>
</rdf:RDF>
