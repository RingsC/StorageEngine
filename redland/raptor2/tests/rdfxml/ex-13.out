_:genid2 <http://somewhere.example.com/base64> "jd8734djr08347jyd4" .
_:genid1 <http://somewhere.example.com/hash-sha-1> _:genid2 .
<http://www.w3.org/> <http://somewhere.example.com/xml-cannonicalized> _:genid1 .
